modules/aufile/aufile_src.c

```diff
@@ -231,7 +231,6 @@ int aufile_src_alloc(struct ausrc_st **stp, const struct ausrc *as,
 	/* return wav format to caller */
 	prm->srate = fprm.srate;
 	prm->ch    = fprm.channels;
-	prm->duration = aufile_get_length(st->aufile, &fprm);
```

Comment:
Is there an advantage to not setting this here? It is very cheap to compute for an opened aufile.